docs/whatsnew.rst

```diff
@@ -12,140 +12,125 @@ New features in eXo Platform 5.2?
-========================================
+:ref:`Create event drawer <CreatingNewEvent>`
 
-**Upgrades**
+Creating a new event has been redesigned as an elegant drawer panel for better usability:
+
+|image6|
+ 
+ 
+:ref:`Badge of new received messages in the mini chat <miniChat>`
 
-eXo Platform 5.1 now supports JBoss EAP 7.1. This new version comes with 
-some `improvements <https://www.redhat.com/en/blog/red-hat-releases-jboss-eap-71>`__ 
-and fixes some issues which impacted eXo Platform:
+When you scroll down into the mini chat discussion and that your interlocutor 
+sends you new chat messages, a badge with the number of unread messages appears 
+next to the display name.
 
-  -  "Remember me" token not recognized in Jboss EAP 7.0 (`PLF-7787 <https://jira.exoplatform.org/browse/PLF-7787>`__)
-  
-  -  Instance id of a cluster node is not added on the JSESSIONID cookie (`PLF-7854 <https://jira.exoplatform.org/browse/PLF-7854>`__)
+|image7|
 
-**Performances and scalability**
+In addition to the above UX improvements, we also:
 
-In eXo Platform 5.1 we continued our effort on improving the performance 
-and the scalability by changing some strategies, configurations and data 
-storage:
+-  Added more spacing at the left and right of the activities for a better readability.
+-  Unified the "Show more" button with the new UX style used accross eXo Platform.
 
--  Users/Groups/Roles import in database: a new job is implemented to 
-   import IDM data  in IDM database asynchronously and periodically.
-  
--  Smarter data user synchronization: only mandatory data are imported 
-   in database during synchronization, other user's data are imported on 
-   login or when it is required (lazy import).
-  
-**Migrations**
+.. _TechnicalNovelties:
 
--  As cited in the previous section, eXo Chat has a new design. 
-   Moreover, from a technical point of view, it has been implemented 
-   using a new frontend framework (Vue.js).
+========================================
+Technical novelties in eXo Platform 5.2
+========================================
 
--  Chat favorites data has also been changed. An upgrade plugin runs on 
-   eXo Platform 5.1 startup to migrate existing data to the new format.
+**Security**
 
-**Some others Improvements**
+In eXo Platform 5.2 we continued our effort on improving the security by 
+changing some permissions and avoiding folder listing through WebDAV.
 
--  Harmonize cache configuration in cluster and local modes.
+**Platform RDBMS**
 
--  Improve Mbean cache naming: All caches are now named with the 
-   following pattern: ``<project>.<cacheName>``
+As in previous versions, we continued to decrease our dependency on JCR 
+to store data. Now, the following components rely on the relational 
+**database** storage for better speed and manageability :
 
--  ``cluster.node.name`` parameter automatically generated when not set.
+-  Product information
+-  Login history
+-  Wiki
+-  Files
+-  Social
+-  Activities and comments
```

Comment:
Activities and comments is included in Social.

Reply:
OK